cob_twist_controller/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN_INCLUDE_DIRS} ${oroco
 set(SRC_C_DIR "src/constraint_solvers")
 set(SRC_CS_DIR "${SRC_C_DIR}/solvers")
 
+add_library(jacobian_derivative src/utils/chainjnttojacdotsolver.cpp)
+add_dependencies(jacobian_derivative ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
+target_link_libraries(jacobian_derivative ${catkin_LIBRARIES})
+
```

Comment:
Well, it's removed now...but where has it been used? Or hasn't it been used at all?